src/daft-connect/src/translation/expr/unresolved_function.rs

```diff
@@ -34,10 +34,13 @@ pub fn unresolved_to_daft_expr(f: &UnresolvedFunction) -> eyre::Result<daft_dsl:
             .wrap_err("Failed to handle >= function"),
         "%" => handle_binary_op(arguments, daft_dsl::Operator::Modulus)
             .wrap_err("Failed to handle % function"),
+        "==" => handle_binary_op(arguments, daft_dsl::Operator::Eq)
+            .wrap_err("Failed to handle == function"),
+        "not" => not(arguments).wrap_err("Failed to handle not function"),
```

Comment:
Maybe put some quotes around "not" or something, otherwise it's not easy to parse the meaning of this error message

Reply:
done